.vscode/launch.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
I wondere if this is something the sbt-daffodil plugin could generate? I think the only thing it wouldn't be able to do is the `"data"` field.

I guess the other issue is SBT see's dependencies are jars and not resource directories, I'm not sure if the VS Code extension would have issues is some schema files were in jars.